internal/commands/user/list_test.go

```diff
@@ -103,158 +99,82 @@ func TestUserListHandler(t *testing.T) {
-		expectedRow      map[string]interface{}
-	}{
-		{
-			description:      "Should show name for apikey type user",
-			authProviderType: realm.AuthProviderTypeAPIKey,
-			user: realm.User{
-				ID:                     "id1",
-				Identities:             []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeAPIKey}},
-				Type:                   "type1",
-				Disabled:               false,
-				Data:                   map[string]interface{}{"name": "myName"},
-				CreationDate:           1111111111,
-				LastAuthenticationDate: 1111111111,
-			},
-			expectedRow: map[string]interface{}{
-				"Enabled":            true,
-				"ID":                 "id1",
-				"Last Authenticated": "2005-03-18 01:58:31 +0000 UTC",
-				"Name":               "myName",
-				"Type":               "type1",
-			},
-		},
-		{
-			description:      "Should show email for local-userpass type user",
-			authProviderType: realm.AuthProviderTypeUserPassword,
-			user: realm.User{
-				ID:                     "id1",
-				Identities:             []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeUserPassword}},
-				Type:                   "type1",
-				Disabled:               false,
-				Data:                   map[string]interface{}{"email": "myEmail"},
-				CreationDate:           1111111111,
-				LastAuthenticationDate: 1111111111,
-			},
-			expectedRow: map[string]interface{}{
-				"Enabled":            true,
-				"ID":                 "id1",
-				"Last Authenticated": "2005-03-18 01:58:31 +0000 UTC",
-				"Email":              "myEmail",
-				"Type":               "type1",
-			},
-		},
-	} {
-		t.Run(tc.description, func(t *testing.T) {
-			assert.Equal(t, tc.expectedRow, userTableRow(tc.authProviderType, tc.user))
-		})
-	}
-}
-
 func TestUserListFeedback(t *testing.T) {
 	for _, tc := range []struct {
-		description    string
-		users          []realm.User
-		expectedOutput string
+		description     string
+		outputs         []userOutput
+		expectedContent string
 	}{
 		{
-			description:    "Should indicate no users found when none are found",
-			users:          []realm.User{},
-			expectedOutput: "01:23:45 UTC INFO  No available users to show\n",
+			description:     "whould indicate no users found when none are found",
+			expectedContent: "01:23:45 UTC INFO  No available users to show\n",
 		},
 		{
-			description: "Should group the users by provider type and sort by LastAuthenticationDate",
-			users: []realm.User{
+			description: "whould group the users by provider type and sort by LastAuthenticationDate",
+			outputs: []userOutput{
 				{
-					ID:                     "id1",
-					Identities:             []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeUserPassword}},
-					Type:                   "type1",
-					Disabled:               false,
-					Data:                   map[string]interface{}{"email": "myEmail1"},
-					CreationDate:           1111111111,
-					LastAuthenticationDate: 1111111111,
+					user: realm.User{
```

Comment:
save a line and move this inline with `outputs: userOutputs{user: realm.User{` (consolidate the end of this declaration to so it ends with `}},`